src/Components/CaseStatusBar/CaseStatusBar.tsx

```diff
@@ -1,10 +1,71 @@
 import React from 'react';
 import { View, Text } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
 
-export default function CaseStatusBar() {
+import styles from './styles';
+import { colors } from '../../styles/colors';
+
+interface CaseStatusBarProps {
+  status: string;
+}
+
+export default function CaseStatusBar({ status }: CaseStatusBarProps) {
+  // stephanie this thing is so mf ugly where should i hide it
+  const getStatusColor = (status: string) => {
```

Comment:
I think it might be a little cleaner in a `switch` statement. We can also move this function to `/src/app/(BottomTabNavigation)/AllCases/utils.ts` because we will also need this for the `CaseCard` component.

Reply:
tried this and it looked even uglier, since switch case has to handle every single possible caseStatus, and since multiple caseStatuses map to the same colors I feel like the if statement makes more sense. Correct me if I'm wrong though!

Reply:
```
export function getCaseStatusColor(status: string): CaseStatusColor {
  switch (status) {
    case 'Pending':
      return {
        background: {
          backgroundColor: colors.lightYellow,
          borderColor: colors.darkYellow,
        },
        text: { color: colors.darkYellow },
      };
    case 'Action Required':
      return {
        background: {
          backgroundColor: colors.lightRed,
          borderColor: colors.darkRed,
        },
        text: { color: colors.darkRed },
      };
    case 'In Progress' ||
      'New Case' ||
      'Settled' ||
      'Appeal' ||
      'Payment Processing' ||
      'Payment Distributed':
      return {
        background: {
          backgroundColor: colors.lightGreen,
          borderColor: colors.darkGreen,
        },
        text: { color: colors.darkGreen },
      };
    default:
      return {
        background: {
          backgroundColor: colors.lightGrey,
          borderColor: colors.midGrey,
        },
        text: { color: colors.darkGrey },
      };
  }
}
```

Reply:
ok, I'll leave it as else if for now LOL